### PR TITLE
Add pinentry package

### PR DESCRIPTION
**Pull request by adrien-delhorme**
Add pinentry, a gnupg dependency package

### CODE REVIEW

File: cross/gnupg/PLIST [hunk 1]
```diff
@@ -12,4 +12,6 @@ bin:bin/gpgsm-gencert.sh
 bin:bin/gpgtar
 bin:bin/gpgv2
 bin:bin/kbxutil
+bin:bin/pinentry
```
Comment (Dr-Bean) [inline cross/gnupg/PLIST]: Don't these two additions belong in the PLIST of `cross/pinentry`?

**Comment by Dr-Bean** [comment]
Thanks. 
One comment on the code. If the gnupg package needs an update, please edit its `SPK_REV` and update the changelog as well. Please confirm that you've tested it (as far as you're able to anyway) :)

**Comment by adrien-delhorme** [comment]
Done.
I've tested on my DS212j: I'm now able to generate keys with `gpg --gen-key`.

File: cross/pinentry/PLIST [hunk 2]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (Dr-Bean) [inline cross/pinentry/PLIST]: This could be cleaned up. Can be done on merge.

File: spk/gnupg/Makefile [hunk 3]
```diff
@@ -9,7 +9,7 @@ MAINTAINER = SynoCommunity
 DESCRIPTION = GnuPG allows to encrypt and sign your data and communication, features a versatile key management system as well as access modules for all kinds of public key directories.
 STARTABLE = no
 DISPLAY_NAME = GnuPG
-CHANGELOG = "Initial release"
+CHANGELOG = "1. Initial release<br>2. Add pinentry dependency"
```
Comment (Dr-Bean) [inline spk/gnupg/Makefile]: Our changelog isn't cumulative, so this should probably be limited to `CHANGELOG = Add pinentry dependency`.
Can be done on merge.

**Pull request merged**